### ISSUE TITLE
Style/IndentationWidth fails with private_class_method

### ISSUE THREAD

**Issue by mseppae** [bbatsov/rubocop#2045]
Rubocop cop Style/IndentationWidth is failing with private_class_method syntax.

The class to test:
```ruby
class Foo
  private_class_method def self.bar
    'bar'
  end
end
```

Error trace:
```
An error occurred while Style/IndentationWidth cop was inspecting /Users/mseppae/foo.rb.
undefined method `line' for nil:NilClass
/Users/mseppae/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/parser-2.2.2.6/lib/parser/source/map.rb:98:in `line'
/Users/mseppae/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/rubocop-0.32.1/lib/rubocop/cop/style/indentation_width.rb:225:in `indentation_to_check?'
/Users/mseppae/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/rubocop-0.32.1/lib/rubocop/cop/style/indentation_width.rb:201:in `check_indentation'
/Users/mseppae/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/rubocop-0.32.1/lib/rubocop/cop/style/indentation_width.rb:97:in `on_send'
(eval):5:in `block (2 levels) in on_send'
```

System information:
```
Mention the following information in the issue report:
0.32.1 (using Parser 2.2.2.6, running on ruby 2.2.0 x86_64-darwin14)
```
